### ISSUE TITLE
[Enhancement] Support building RHEL-6 guide with new openscap

### ISSUE THREAD

**Issue by iankko** [OpenSCAP/scap-security-guide#554]
Newer openscap versions (those with rewritten HTML guide / report functionality)  require profile name to be specified in order to particular rules description to be included. The RHEL/6's 'make guide' Makefile section contains the following snippet:

```
 44 guide: shorthand2xccdf
 45 #       remove auxiliary Groups which are only for use in tables, and not guide output.
 46 #       specifying a nonexistent profile, "allrules," to make oscap print all Rules
 47         xsltproc -o $(OUT)/unlinked-$(PROD)-xccdf-guide.xml $(TRANS)/xccdf-removeaux.xslt $(OUT)/unlinked-$(PROD)-xccdf.xml
 48         xsltproc -o $(OUT)/unlinked-notest-$(PROD)-xccdf-guide.xml $(TRANS)/xccdf-removetested.xslt $(OUT)/unlinked-$(PROD)-xccdf-guide.xml
 49         oscap xccdf generate guide --profile allrules $(OUT)/unlinked-notest-$(PROD)-xccdf-guide.xml > $(OUT)/$(PROD)-guide.html
```

Since the 'allrules' specifies non-existing profile, this won't work with newer openscap versions.
Therefore enhance the RHEL-6 'make guide' target content it to support building the guide with both openscap versions (openscap both with old and new way of generating HTML guide / report).

**Comment by mpreisler** [comment]
This will be solved as part of #578

**Comment by redhatrises** [comment]
With the merge of #578, I believe that this has been fixed.
